Undefined name 's' in estimate_gradient_norm.py

[flake8](http://flake8.pycqa.org) testing of https://github.com/IBM/CLEVER-Robustness-Score on Python 3.7.0

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./utils.py:31:11: F821 undefined name 'Image'
    pic = Image.fromarray(fig)
          ^
./estimate_gradient_norm.py:467:53: F821 undefined name 's'
                        s12_2_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], axis = 1)
                                                    ^
./estimate_gradient_norm.py:467:75: F821 undefined name 's'
                        s12_2_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], axis = 1)
                                                                          ^
./estimate_gradient_norm.py:468:53: F821 undefined name 's'
                        s12_1_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], ord=1, axis = 1)
                                                    ^
./estimate_gradient_norm.py:468:75: F821 undefined name 's'
                        s12_1_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], ord=1, axis = 1)
                                                                          ^
./estimate_gradient_norm.py:469:53: F821 undefined name 's'
                        s12_i_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], ord=np.inf, axis = 1)
                                                    ^
./estimate_gradient_norm.py:469:75: F821 undefined name 's'
                        s12_i_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], ord=np.inf, axis = 1)
                                                                          ^
7     F821 undefined name 's'
7
```

--- a/estimate_gradient_norm.py
+++ b/estimate_gradient_norm.py
@@ -463,7 +463,9 @@ class EstimateLipschitz(object):
                             feed_dict = {self.img: batch_inputs, self.target_label: target_label, self.true_label: true_label})
                 
                     if self.compute_slope:
-                        # compute distance between consecutive samples: not use sequential samples 
+                        # compute distance between consecutive samples: not use sequential samples
+                        # 's' is undefined in the nex three lines.
+                        # This is a dead code for internal testing and will not affect the CLEVER score computation.
                         s12_2_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], axis = 1)
                         s12_1_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], ord=1, axis = 1)
                         s12_i_norm = np.linalg.norm(s[0:batch_size-1:2] - s[1:batch_size:2], ord=np.inf, axis = 1)

